cfme/tests/containers/test_basic_metrics.py

```diff
@@ -18,7 +21,7 @@ def test_basic_metrics(provider):
     routes = provider.mgmt.o_api.get('route')[1]['items']
     metrics_hostname = [route for route in routes
                         if route['metadata']['name'] == 'hawkular-metrics']
-    assert metrics_hostname, 'Could not find route for hawkular-metrics'
+    assert metrics_hostname, 'Could not find route for metrics'
     metrics_hostname = metrics_hostname.pop()['spec']['host']
     host_url = 'https://' + metrics_hostname + '/hawkular/metrics/'
```

Comment:
This test requires a massive code refactoring, IMO instead of using the ```curl``` command, we can use python's ```request``` module. In addition, to validate Prometheus using ```curl``` do like the following example:
```curl -H "Authorization: Bearer <OCP token>" -k https://<Prometheus metrics route>/graph```.